webapp-django/crashstats/crashstats/templates/crashstats/ratelimit_blocked.html

```diff
@@ -0,0 +1,32 @@
+{% extends "crashstats_base.html" %}
```

Comment:
If you're rate-limiting the ``search_results`` view, you will likely want to remove this. Otherwise, it will load the entire crash-stats UI inside a div in Super Search, leading to a very weird result. 

That seems a bit weird though, as that makes this view a lot less generic, so maybe you'll want to take a different approach to this?